server/package.json

```diff
@@ -4,11 +4,19 @@
   "description": "node node node",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start-server": "node index.js"
```

Comment:
why do not people want to install a npm?

Reply:
There are two types of `npm install`, right. With the global option `npm install -g __some__module__` or without `-g`. If you install a module globally, you can use it on every project. 
To be clear, modules that were installed globally, you can use everywhere on Terminal. 
And the modules that were installed for a project without `-g`, you can ONLY use via the commands you specified on `package.json`.

But, you probably have `nodemon` installed globally, which means that you can use `nodemon` without having to keep it in `package.json`. This requires additional communication to ask team to install a certain package globally, otherwise, it's not gonna work. 

> why do not people want to install a npm?

So, people will install `npm install` that will install all the specified modules. But people won't automatically do `npm install -g nodemon`, which is outside of the project's responsibility. 

So we need not to consider the global install options to make it clear, that they will get all modules just by doing `npm install`. and because it's the project's dependency, not global, we need to provide the command, which was the original point of this comment.